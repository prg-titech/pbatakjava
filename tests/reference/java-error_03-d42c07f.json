{
    "basename": "java-error_03-d42c07f",
    "cmd": "{infile}",
    "infile": "tests/error_03/build.sh",
    "infile_hash": "02de591ee2868952f1fa8b0d23cef3bc4f7e51b636567a6d3c05b60d",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "java-error_03-d42c07f.stderr",
    "stderr_hash": "e2a52fe0de58540c7a8ce8aadba15b01ac184e3bf6c00bcc0ab914bb",
    "returncode": 4
}