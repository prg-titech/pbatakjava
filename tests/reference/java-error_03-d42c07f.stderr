tests/error_03/Test.java:5: error: no applicable method for update(new Point(2, 2))
tests/error_03/Test.java:6: error: no applicable method for m()
Solution not found.
